nifi-assembly/pom.xml

```diff
@@ -363,147 +363,6 @@ language governing permissions and limitations under the License. -->
         </dependency>
     </dependencies>
 
-    <properties>
```

Comment:
@JPercivall rebuilding from top now, I did check and am pretty sure that nifi.properties still winds up in nifi-assembly.

If you meant that it's not in the toolkit assembly, that's correct, it is built into the ssl toolkit jar.  The ssl toolkit is capable of generating a nifi.properties updated with the correct ssl values.  If the user does not specify a base file to use, it uses the embedded one.